demo/app/views/layouts/component_preview.html.erb

```diff
@@ -33,5 +33,7 @@
         </div>
       <% end %>
     </div>
+    <%# TODO: Use live-region-element %>
+    <div aria-live="polite" aria-atomic="true" class="sr-only"> 
```

Comment:
I'm thinking we should embed [live-region-element](https://github.com/primer/live-region-element/) within the lookbook preview once it's available!